### DISCUSSION TITLE
In the "Motoya" font, I propose to increase the contours of the number "4" by 0.5 millimeters.

### DISCUSSION THREAD

**Discussion by Yurij20** [PE5PVB/TEF6686_ESP32#495] [Ideas]
In the "Motoya" font, I propose to increase the contours of the number "4" by 0.5 millimeters.  It just seems that its contours look too thin compared to other numbers.  For example, thicken it to the contours of the number “1”

**Comment by PE5PVB** [comment]
We updated this font. Please check.

**Comment by Yurij20** [comment]
Thank you.  I'm waiting for the next firmware release, then I'll check)

**Comment by Yurij20** [comment]
Everything works as it should, thanks
